TheraLang.Web/ClientApp/src/app/modules/cms-generic/components/piranha-page/piranha-page.component.ts

```diff
@@ -12,12 +12,12 @@ export class PiranhaPageComponent implements OnInit {
   page: Page;
   @Input() cmsRoute: CmsRoute;
   ifGenerate = false;
-  constructor(private cmsPageService: CmsPageService) {}
+  constructor(private cmsPageService: PageService) {}
 
   async ngOnInit() {
-    this.page = await this.cmsPageService.getPiranhaPageById(
-      parseInt(this.cmsRoute.id)
-    );
+    // this.page = await this.cmsPageService.getPageById(
+    //   parseInt(this.cmsRoute.id)
+    // );
```

Comment:
We could remove cms-generic folder completely

Reply:
> We could remove cms-generic folder completely

yes, but I think it is better to create a separate pr, where components, services and everything connected with cms would be removed.
my pr about refactoring page service :)

Reply:
just create a ticket and add TODO here with nubmer of a ticket